iina/MainWindowController.swift

```diff
@@ -2265,6 +2262,16 @@ class MainWindowController: PlayerWindowController {
     }
   }
 
+  private func refreshSeekTimeAndThumnail(from event: NSEvent) {
```

Comment:
Would you mind renaming it without a typo?
```suggestion
  private func refreshSeekTimeAndThumbnail(from event: NSEvent) {
```

Reply:
Pushed another commit which should address feedback, thanks.